docs/administering/for-work.md

```diff
@@ -56,8 +56,8 @@ Implementation notes for LDAP that may apply to your site:
 
 SAML 2.0 is a passwordless single sign-on protocol. It allows a web application such as Sandstorm to
 request the current user's credentials from a central service, typically administered by a
-university or corporate IT team. We expect Sandstorm's SAML support to be compatible with
-Shibboleth, Okta, SimpleSAMLphp, Microsoft ActiveDirectory, and many other systems.
+university or corporate IT team. Sandstorm's SAML support supports Shibboleth, Okta, SimpleSAMLphp,
```

Comment:
Also, "support supports" just looks weird.  How about:

Sandstorm's SAML login works with Shibboleth, Okta, Microsoft Active Directory, SimpleSAMLphp, and other SAML services.